utils.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+"""
+Helper functions
+"""
+from _socket import timeout
+
+import paramiko
+
+
+def is_service_in_status(ssh, name, expected_status):
+    """Helper function for check_services_running to check a specific service's status"""
+    stdin, stdout, stderr = ssh.exec_command("systemctl status {}".format(name))
+    output = stdout.read()
+    status = output.decode('utf-8').strip()
+    return expected_status in status
+
+
+def host_ips(host_service):
+    """
+    Helper function for check_services_running to find an ip of the host to connect to.
+    This is a workaround of the following issues in Paramiko library:
+    https://github.com/paramiko/paramiko/issues/686
+    https://github.com/paramiko/paramiko/issues/770
+    """
+    nics = host_service.nics_service().list()
+    # return only ips 'startswith("10.")' to catch local network ipv4
+    return [n.ip.address for n in nics if n.ip.address.startswith("10.")]
+
+
+def ssh_client(host_service, username, password):
+    """
+    SSH client with a workaround for using IPv4 addresses
+    """
+    ip_addresses = host_ips(host_service)
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    for ip in ip_addresses:
+        try:
+            ssh.connect(ip, username=username, password=password)
+            break
+        except timeout:
+            continue
```

Comment:
**REQUIRED** I tried running this with a host that will timeout, and it was taking forever to timeout. So let's pass `timeout=60` in ssh.connect() so it will timeout predictably. 

Ref: http://docs.paramiko.org/en/latest/api/client.html

can you think of a way to handle timeout better( maybe with some flag), so if all the available addresses result in `timeout` then we should raise that error. If we don't do that it fails like this(with AttributeError):

```
[2019-07-05 11:38:28,939 check_rhv_main.py INFO   ] Calling check check_services_running
[2019-07-05 11:38:42,043 check_rhv_main.py ERROR  ] Exception occurred during execution of check_services_running
Traceback (most recent call last):
  File "check_rhv_main.py", line 100, in main
    measure_func(system, **json.loads(args.services.replace("'", "\"")))
  File "/home/kkulkarn/git/infra/check-rhv/rhv_checks.py", line 355, in check_services_running
    service = is_service_in_status(ssh, service_name, status)
  File "/home/kkulkarn/git/infra/check-rhv/utils.py", line 12, in is_service_in_status
    stdin, stdout, stderr = ssh.exec_command("systemctl status {}".format(name))
  File "/home/kkulkarn/git/infra/check-rhv/env/lib/python2.7/site-packages/paramiko/client.py", line 508, in exec_command
    chan = self._transport.open_session(timeout=timeout)
AttributeError: 'NoneType' object has no attribute 'open_session'

```